apps/prairielearn/src/pages/instructorAssessmentAccessOverrides/instructorAssessmentAccessOverrides.ejs

```diff
@@ -0,0 +1,275 @@
+								<% if (!assessment.group_work) { %>
+									<div class="form-group">
+										<label for="student_uid">Student UID</label>
+										<input type="text" class="form-control" id="student_uid" name="student_uid" placeholder="student@example.com" required>
+									</div>
+								<% } else { %>
+									<div class="form-group">
+										<label for="group_id">Group Name</label>
+										<input type="text" class="form-control" id="group_name" name="group_name" required>
+									</div>
+								<% } %>
+								<div class="form-group">
+									<label for="start_date">Start Date</label>
+									<input type="datetime-local" class="form-control" id="start_date" name="start_date" required>
+								</div>
+								<div class="form-group">
+									<label for="end_date">End Date</label>
+									<input type="datetime-local" class="form-control" id="end_date" name="end_date" required>
+									<div id="end_date_error" class="invalid-feedback" style="display: none; color: red;"></div>
+								</div>
+								<div class="form-group">
+									<label for="credit">Credit</label>
+									<div class="input-group">
+									<input type="number" class="form-control" id="credit" name="credit" min="0" required>
+									<div class="input-group-append">
+										<span class="input-group-text">%</span>
+									</div>
+								  </div>
+								</div>
+								<div class="form-group">
+									<label for="note">Note</label>
+									<textarea class="form-control" id="note" name="note"></textarea>
+								</div>
+						</div>
+						<div class="modal-footer">
+							<button type="button" class="btn btn-secondary" data-dismiss="modal">Close</button>
+							<button type="submit" class="btn btn-primary">Submit</button>
+						</div>
+						</form>
+					</div>
+				</div>
+			</div>
+			<div class="modal fade" id="editPolicyModal" tabindex="-1" role="dialog" aria-labelledby="editPolicyModalLabel" aria-hidden="true">
+				<div class="modal-dialog" role="document">
+					<div class="modal-content">
+						<div class="modal-header">
+							<h5 class="modal-title" id="editPolicyModalLabel">Edit Override</h5>
+							<button type="button" class="close" data-dismiss="modal" aria-label="Close">
+								<span aria-hidden="true">&times;</span>
+							</button>
+						</div>
+						<div class="modal-body">
+							<form id="edit-override-form" method="POST">
+								<input type="hidden" name="__action" value="edit_override">
+								<input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+								<input type="hidden" name="policy_id" id="policy_id">
+								
+								<% if (!assessment.group_work) {  %>
+								<div class="form-group">
+									<label for="student_uid">Student UID</label>
+									<input type="text" class="form-control" id="edit-student_uid" name="student_uid">
+								</div>
```

Comment:
The bodies of if/else blocks in EJS should be indented.

Reply:
done in ce8a17fa8